couchdb/server/__main__.py

```diff
@@ -47,6 +47,11 @@
   --egg-cache=<path>      specifies egg cache dir. If omitted, PYTHON_EGG_CACHE
                           environment variable value would be used or system
                           temporary directory if variable not setted.
+  --couchdb-version=<ver> define with which version of couchdb server will work
+                          default: latest implemented.
+                          Supports from 0.9.0 to 1.1.0 and trunk. Technicaly
+                          should work with 0.8.0.
+                          e.g.: --couchdb-version=0.9.0
```

Comment:
Okay. To be honest, i'm lack of Erlang skill. Please excuse me for simply praying that those code related to older protocol work fine.